pkgs/development/python-modules/pygame/default.nix

```diff
@@ -64,6 +64,8 @@ buildPythonPackage rec {
   ];
 
   postPatch = ''
+    substituteInPlace buildconfig/config_{unix,darwin}.py \
+      --replace-fail 'from distutils' 'from setuptools._distutils'
```

Comment:
Should we use this patch instead?
https://github.com/pygame/pygame/commit/6038e7d6583a7a25fcc6e15387cf6240e427e5a7